python/paddle/distributed/checkpoint/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .save_state_dict import save_state_dict
+from .load_state_dict import load_state_dict
+
+__all__ = [
+    "save_state_dict",
+    "load_state_dict",
+]
```

Comment:
Only add API in list of __ all__ at recommended user path, as we recommend using `paddle.distributed.save_state_dict` and `paddle.distributed.load_state_dict`, there is no need to add them to this list. `import` above can be retained.

Reply:
ok